sdk/cpp/ydk/CMakeLists.txt

```diff
@@ -44,34 +44,36 @@ set(libydk_src src/netconf_provider.cpp src/netconf_client.cpp src/value.cpp src
 set(libydk_headers src/netconf_provider.hpp src/netconf_client.hpp src/core.hpp src/entity_data_node_walker.hpp src/types.hpp src/errors.hpp)
 set(libydk_install_headers src/netconf_provider.hpp src/core.hpp src/crud_service.hpp src/errors.hpp src/types.hpp)
 
+add_library(temp OBJECT ${libydk_src} ${libydk_headers})
```

Comment:
Can we call this library something other than 'temp'.. something like 'dependencies'?